analyses/DESeq_analysis/run-DESeq-analysis.R

```diff
@@ -0,0 +1,195 @@
+# Author: Sangeeta Shukla and Alvin Farrel
+# Date: June 2021
+# Function: 
+# 1. summarize Differential expression from RNASeq data
+# 2. tabulate corresponding P-value
+
+# Example run: DESeq
+# Rscript analyses/DESeq/run-DESeq-analysis.R 1 1
+# This was compares 1 cancer group (Combined or Cohort specific) with one GTEx tissue type. The input arguments are the indices of the comparison of the cancer groups and gtex sub tissues to be compared. In v6, there are 107 cancer groups vs 54 GTEx tissues
+
+# Read arguments from terminal
+args <- commandArgs(trailingOnly = TRUE)
+
+# Load required libraries
+suppressPackageStartupMessages({
+  library(ggplot2)
+  #library(DESeq2)
+})
+
+
+HIST_index <- args[1]   #assign first argument to Histology index
+GTEX_index <- args[2]   #assign second argument to GTEx index
+
+    
+
+
+
+#Load histology file
+hist <- read.delim("histologies.tsv", header=TRUE, sep = '\t')
+
+#Load expression counts data
+countData <- readRDS("gene-counts-rsem-expected_count-collapsed.rds")
+
+#Load expression TPM data
+TPMData <- readRDS("gene-expression-rsem-tpm-collapsed.rds")
+
+#Load EFO-MONDO map file
+EFO_MONDO <- read.delim("efo-mondo-map.tsv", header =T)
+
+#Load gene symbol-gene ID RMTL file
+ENSG_Hugo <- read.delim("ensg-hugo-rmtl-v1-mapping.tsv", header =T)
```

Comment:
Thanks for the comments here @logstar. Just to piggy back onto this, the code should be written with your analysis module directory as your current directory, per the readme [here](https://github.com/PediatricOpenTargets/OpenPedCan-analysis#folder-structure). Please update the paths for this convention so that all modules can be run in the same manner.

Reply:
@jharenza @logstar I have updated the code to run with analyses directory as current directory.
Will add README.md next.